docs/src/pages/[platform]/components/heading/examples/HeadingThemeExample.tsx

```diff
@@ -0,0 +1,30 @@
+import { Heading, Flex, ThemeProvider, Theme } from '@aws-amplify/ui-react';
+
+const theme: Theme = {
+  name: 'heading-theme',
+  tokens: {
+    components: {
+      heading: {
+        color: { value: '{colors.blue.80}' },
```

Comment:
Switch to another color maybe? The contrast ratio is not high enough in dark mode
![Screen Shot 2022-06-15 at 3 06 29 PM](https://user-images.githubusercontent.com/40295569/173940958-178f87f9-0e5f-4dfb-9fa6-2f0ef6b8b2b7.png)